shopfloor_reception/__manifest__.py

```diff
@@ -1,7 +1,7 @@
 {
     "name": "Shopfloor Reception",
     "summary": "Reception scenario for shopfloor",
-    "version": "14.0.2.3.0",
+    "version": "14.0.2.4.0",
```

Comment:
the migration script must have the same version

Reply:
changed